ledger-tool/src/bigtable.rs

```diff
@@ -1061,6 +1092,16 @@ pub fn bigtable_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
             };
             runtime.block_on(block(slot, output_format, config))
         }
+        ("entries", Some(arg_matches)) => {
+            let slot = value_t_or_exit!(arg_matches, "slot", Slot);
+            let config = solana_storage_bigtable::LedgerStorageConfig {
+                read_only: false,
```

Comment:
I see some of the other methods that only fetch (`block`, blocks`) also use `read_only == false`. We should be ok to run this with `read_only == true` though right? Or maybe there is some subtlety that requires R/W too

Reply:
Bad c/p, I think. `read_only: true` should be sufficient. I will verify and change.

Reply:
I'll also audit the others and PR separately if there are some that can be downgraded to read-only.